include/mbedtls/ecp_internal.h

```diff
@@ -98,7 +104,7 @@ int mbedtls_internal_ecp_init( const mbedtls_ecp_group *grp );
  */
 void mbedtls_internal_ecp_free( const mbedtls_ecp_group *grp );
 
-#if defined(ECP_SHORTWEIERSTRASS)
+/* Short Weierstrass form */
```

Comment:
I'm not really sure why removing these `ECP_SHORTWEIERSTRASS` and `ECP_MONTGOMERY` ifdefs causes ECP functions with ALT implementations to be included in the Doxygen output. If why isn't obvious to an informed developer, could you please explain why in the commit message?

Reply:
I believe this will also fix #1147

Reply:
I've added a clarifying description to the commit. Please let me know if it's sufficient.